CONTRIBUTING.md

```diff
@@ -37,15 +38,7 @@ From a separate terminal, start running a flutter app to connect to DevTools:
 
 ## Development
 
-To run DevTools as a Flutter web app, follow these steps.
-
-First, perform one-time setup:
-
-- `flutter config --enable-web`
-
-Now you can run the app at any time with the flutter command.
-
-From the packages/devtools_app directory:
+To run DevTools as a Flutter web app, from the packages/devtools_app directory:
```

Comment:
Can you add a note above this line that says, "*NOTE:* Though DevTools is shipped as a Flutter Web app, we recommend developing as a Flutter Desktop app where possible for a more efficient development workflow. Please see the [Desktop Embedder] section below for instructions on running DevTools as a Flutter Desktop app."

Reply:
done